arc/aws/391835788720/us-east-1/main.tf

```diff
@@ -6,7 +6,7 @@ terraform {
       source  = "hashicorp/random"
     }
     aws    = {
-      version = ">= 6.0"
```

Comment:
This feels very unrelated to PR title. Why is this needed? Why do we want to downgrade AWS version?

Reply:
Yes we need to downgrade as I discovered 6.0 was only released last week and many Terraform modules do not work with it yet. My change makes it match the requirements of the EKS module which is what we want to deploy.

https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/versions.tf

This is a low risk change since this Terraform stack is new. Nothing production uses it yet.